plugins/outputs/kinesis/README.md

```diff
@@ -13,16 +13,42 @@ maybe useful for users to review Amazons official documentation which is availab
 
 This plugin uses a credential chain for Authentication with the Kinesis API endpoint. In the following order the plugin
 will attempt to authenticate.
+
 1. Assumed credentials via STS if `role_arn` attribute is specified (source credentials are evaluated from subsequent rules)
 2. Explicit credentials from `access_key`, `secret_key`, and `token` attributes
 3. Shared profile from `profile` attribute
 4. [Environment Variables](https://github.com/aws/aws-sdk-go/wiki/configuring-sdk#environment-variables)
 5. [Shared Credentials](https://github.com/aws/aws-sdk-go/wiki/configuring-sdk#shared-credentials-file)
 6. [EC2 Instance Profile](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html)
 
+## Example Configuration
+
+```toml
+  access_key = "AWSKEYVALUE"
+  secret_key = "AWSSecretKeyValue"
+  region = "eu-west-1"
+  streamname = "KinesisStreamName"
+  aggregate_metrics = true
+  # Either "gzip", "snappy"
+  compress_metrics_with = gzip
```

Comment:
should be ` compress_metrics_with = "gzip" `

Reply:
will fix